### PR TITLE
[WIP] Graduation blog post stub

### PR DESCRIPTION
**Pull request by juliusv**
Signed-off-by: Julius Volz <julius.volz@gmail.com>

### CODE REVIEW

**Comment by juliusv** [comment]
@RichiH I created the stub page for you - now you just need to fill in the details :)

**Comment by RichiH** [comment]
Thanks!

**Comment by RichiH** [comment]
@juliusv PTAL

File: content/blog/2018-08-09-prometheus-graduates-within-cncf.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+We are happy to announce that as of today, Prometheus graduates within CNCF.
+
+Prometheus is the second project ever to make it to this tier.
+By graduating Prometheus, CNCF shows that it's confident in our code and feature velocity, our maturity and stability, and our governance and community processes.
+This also acts as an external verification of quality for anyone needing this for their internal Q&A.
+
+Since reaching incubation level, a lot of things happened; some of which stand out:
+
+* We redid our storage back-end to support high churn in services
+* We had a large push towards stability, with 2.3.2 having no known bugs or issues
```
Comment (grobie) [inline content/blog/2018-08-09-prometheus-graduates-within-cncf.md]: Uuuhm, SoundCloud still needs to regularly downgrade servers due to service discovery getting stuck. There are more reports from others. Sounds like a very bold claim.

File: content/blog/2018-08-09-prometheus-graduates-within-cncf.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+* We started a documentation push with a special focus on making Prometheus adoption and joining the community easier
+
+Especially the last point is important as we currently enter our fourth phase of adoption. These phases were adoption by
+
+# Monitoring-centric users actively looking for the very best in monitoring
```
Comment (grobie) [inline content/blog/2018-08-09-prometheus-graduates-within-cncf.md]: Is this supposed to be an ordered list? Use numbers instead (repeating `1.` works).

File: content/blog/2018-08-09-prometheus-graduates-within-cncf.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: Prometheus Graduates Within CNCF
+created_at: 2018-08-09
+kind: article
+author_name: Richard Hartmann
+---
+
+We are happy to announce that as of today, Prometheus graduates within CNCF.
```
[the remark this answers is not in the document]
Comment (brian-brazil) [reply content/blog/2018-08-09-prometheus-graduates-within-cncf.md]: within the CNCF

File: content/blog/2018-08-09-prometheus-graduates-within-cncf.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+Prometheus is the second project ever to make it to this tier.
+By graduating Prometheus, CNCF shows that it's confident in our code and feature velocity, our maturity and stability, and our governance and community processes.
+This also acts as an external verification of quality for anyone needing this for their internal Q&A.
```
Comment (brian-brazil) [inline content/blog/2018-08-09-prometheus-graduates-within-cncf.md]: "internal discussions around choice of monitoring tool" might read a bit better

File: content/blog/2018-08-09-prometheus-graduates-within-cncf.md (hunk 1 above)
Comment (brian-brazil) [reply content/blog/2018-08-09-prometheus-graduates-within-cncf.md]: 2.3.2 has one known bug

File: content/blog/2018-08-09-prometheus-graduates-within-cncf.md [hunk 5]
```diff
@@ -5,23 +5,23 @@ kind: article
...
 By graduating Prometheus, CNCF shows that it's confident in our code and feature velocity, our maturity and stability, and our governance and community processes.
-This also acts as an external verification of quality for anyone needing this for their internal Q&A.
+This also acts as an external verification of quality for anyone in internal discussions around choice of monitoring tool.
 
 Since reaching incubation level, a lot of things happened; some of which stand out:
 
-* We redid our storage back-end to support high churn in services
-* We had a large push towards stability, with 2.3.2 having no known bugs or issues
+* We completely rewrote our storage back-end to support high churn in services
+* We had a large push towards stability, with 2.3.2 having only onw known bug
```
Comment (grobie) [inline content/blog/2018-08-09-prometheus-graduates-within-cncf.md]: s/onw/one

**Comment by grobie** [comment]
> 2.3.2 has one known bug

Hm, looking at https://github.com/prometheus/prometheus/issues?q=is%3Aissue+is%3Aopen+label%3Akind%2Fbug, what do you consider a bug @brian-brazil? The stale discovery (assuming k8s and ec2 suffer from the same underlying issue) and consul connection leak are already two. 

What about not getting into that game at all and changing the statement to something like ` We had a large push towards stability with our latest v2.3.2 release`.

**Comment by brian-brazil** [comment]
I was thinking of the rules status page issue, your suggestion makes sense.

**Pull request merged**